locations/spiders/dollar_tree_us.py

```
@@ -1,19 +1,23 @@
+from scrapy.spiders import SitemapSpider
+
 from locations.categories import Categories, apply_category
 from locations.items import Feature
-from locations.storefinders.yext_answers import YextAnswersSpider
+from locations.structured_data_spider import StructuredDataSpider
 
 
-class DollarTreeUSSpider(YextAnswersSpider):
+class DollarTreeUSSpider(SitemapSpider, StructuredDataSpider):
     name = "dollar_tree_us"
     item_attributes = {"brand": "Dollar Tree", "brand_wikidata": "Q5289230"}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)